### ISSUE TITLE
[ui5-popover]: ACC-263.1- Dialog "role" is not recognized by jaws Screen reader.

### ISSUE THREAD

**Issue by EranLavi** [SAP/ui5-webcomponents#6845]
### Bug Description

ui5-popover dialog in not recognized as a dialog by screen reader jaws.

### Affected Component

ui5-popover

### Expected Behaviour

Screen reader should announce like "Newsletter subscription" dialog.

### Isolated Example

https://codesandbox.io/s/ui5-webcomponents-forked-18qnlr?file=/index.html

### Steps to Reproduce

1.open https://codesandbox.io/s/ui5-webcomponents-forked-18qnlr?file=/index.html
2. open "Jaws" screen reader
3. click on "open popover" button.
4. Screen reader should announce like "Newsletter subscription" dialog.!!! acc-263.1 error

### Priority

Medium

### UI5 Web Components Version

1.11.0

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by TeodorTaushanov** [comment]
Hi @EranLavi,

I'm testing your example with JAWS 2023 and the reading seem correct. Here is the JAWS speech history:

```
Open Popover Emphasized  Button  
To activate press Enter.
Newsletter subscription  dialog  
Email:
Enter Email  Edit  
```

Please, when you're testing the screen readers reading don't use mouse click (as it is not a real usage of screen readers), but navigate to the button and press keyboard Enter button instead. 

Best,
Teo
